estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,39 @@
+<odoo>
+    <record id="estate_property_offers_tree_view" model="ir.ui.view">
+        <field name="name">estate.property.offers.tree</field>
+        <field name="model">estate.property.offer</field>
+        <field name="arch" type="xml">
+            <tree string="Property Offers" editable="bottom">
+                <field name="price"/>
+                <field name="partner_id"/>
+                <field name="validity"/>
+                <field name="date_deadline"/>
+                <button name="set_offer_to_accepted" string=" " type="object" icon="fa-check" attrs="{'invisible': ['|',('status', '=', 'accepted'),('status', '=', 'refused')]}"/>
+                <button name="set_offer_to_refused" string=" " type="object" icon="fa-times" attrs="{'invisible': ['|',('status', '=', 'accepted'),('status', '=', 'refused')]}"/>
```

Comment:
combining the conditions of the domain to enhance readibility (status in [accepted, refused])